Cargo.toml

```diff
@@ -73,9 +68,14 @@ members = [
   "crates/db_views/report_combined",
   "crates/db_views/search_combined",
   "crates/db_views/site",
-  "crates/routes",
+  "crates/api/api",
+  "crates/api/api_crud",
+  "crates/api/api_common",
+  "crates/api/api_utils",
+  "crates/apub",
+  "crates/apub_objects",
   "crates/federate",
-  "crates/email",
+  "crates/routes",
 ]
```

Comment:
Reordered these so that crates which are faster to compile come first, which helps with testing the analyzer.